README.md

```diff
@@ -4,6 +4,16 @@ Dockerized Ethereum testnets
 
 - - -
 
+## Table of Contents
+
+* [Requirements](#requirements)
+* [Network up](#network-up)
+* [Network topology and configuration](#network-topology-and-configuration)
+* [Working with nodes from docker host](#working-with-nodes-from-docker-host)
+* [Working with a node from its container](#working-with-a-node-from-its-container)
```

Comment:
@olawiberg : I think it's worth also putting a link to the example over here -- approving PR but consider adding example to table of contents before merging.